src/Util.java

```diff
@@ -0,0 +1,16 @@
+public class Util {
+
+    public static int isInteger( String input, Player player ) {
+        try {
+
+            int val = Integer.parseInt( input );
+            if(player != null){
```

Comment:
If you use exception, you don't need to check the validity of the object.

If the object does not exist, you can return error anyway.